21948 - Change Continuation Application

ArwenQin opened the pull request:
*Issue #:* /bcgov/entity#21948

*Description of changes:*
- Changed "Continuation In" to "Continuation Application" in create UI

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

ArwenQin commented:
Create UI after change:
![image](https://github.com/bcgov/business-create-ui/assets/122495122/f61fed49-fe1e-43ea-bf3a-685adacaa408)

ArwenQin commented:
/gcbrun

bcregistry-sre commented:
Temporary Url for review: https://business-create-dev--pr-721-t3j7ql6o.web.app

severinbeauvais commented:
Please rebase so that you pick up Karim's latest config change, and check whether the testing-coverage error is now fixed.

severinbeauvais reviewed (approved):
Looks good. Don't merge until we get Andy's thoughts on the NR request type label.

ArwenQin commented on src/components/common/NameRequestInfo.vue:
```diff
@@ -306,7 +306,7 @@ export default class NameRequestInfo extends Mixins(CommonMixin, DateMixin) {
       case NrRequestActionCodes.RESTORE: return 'Restoration Request'
       case NrRequestActionCodes.AMALGAMATE: return 'Amalgamation'
-      case NrRequestActionCodes.MOVE: return 'Continuation In'
+      case NrRequestActionCodes.MOVE: return 'Continuation Application'
```
Confirmed with Andy, we will leave this one as 'Continuation In'

ArwenQin commented:
> Looks good. Don't merge until we get Andy's thoughts on the NR request type label.

Thanks you Severin. I just confirmed with Andy and changed the name request code back.
Rebased.
Could you please double check if we can merge now? Thank you.

bcregistry-sre commented:
Temporary Url for review: https://business-create-dev--pr-721-t3j7ql6o.web.app
